backend/oasst_backend/user_stats_repository.py

```diff
@@ -39,6 +39,7 @@ def get_leaderboard(self, time_frame: UserStatsTimeFrame, limit: int = 100) -> L
             self.session.query(User.id.label("user_id"), User.username, User.auth_method, User.display_name, UserStats)
             .join(UserStats, User.id == UserStats.user_id)
             .filter(UserStats.time_frame == time_frame.value)
+            .filter(User.stats_enabled)
```

Comment:
does this really do the correct thing, or should it be `== True`?

Reply:
yes, I think single boolean var works. Problem is always that linter things `abc == True` is evil ... but `abc is True` doesn't work with sqlalchemy (!) .. so you could use `sa.true()` for example to make linter happy.
Multiple filter conditions (AND) can be specified on one `filter(conditionA, conditionB)` call with comma separated...